source/Engine.h

```diff
@@ -261,8 +261,10 @@ class Engine {
 	TestContext *testContext = nullptr;
 
 	double zoom = 1.;
+	double baseZoom = 1.;
 	// Tracks the next zoom change so that objects aren't drawn at different zooms in a single frame.
 	double nextZoom = 0.;
+	double zoomMod = 2.;
```

Comment:
I know this gets set before it's used, but it feels weird to me for the "default" to be 2.
```suggestion
	double zoomMod = 1.;
```